docs/fsharp/style-guide/formatting.md

```diff
@@ -646,14 +646,21 @@ if a then
 if a then ()
 ```
 
-If any of the expressions are multi-line or `if/then/else` expressions.
+If any of the expressions are multi-line, each conditional branch should be multi-line.
```

Comment:
I wasn't sure if I should update this for both of these cases, since they're essentially describing the same thing, one just includes some `elif` branches, so I aired on the side of including both.